io.openems.edge.controller.evcs/src/io/openems/edge/controller/evcs/EvcsController.java

```diff
@@ -104,7 +115,8 @@ void activate(ComponentContext context, Config config) throws OpenemsNamedExcept
 		this.defaultChargeMinPower = Math.max(0, config.defaultChargeMinPower());
 		this.chargeMode = config.chargeMode();
 		this.priority = config.priority();
-		this.evcsId = config.evcs_id();
+		this.ess = this.componentManager.getComponent(config.ess_id());
+		this.evcs = this.componentManager.getComponent(config.evcs_id());
```

Comment:
This should always be done in the run()-method, because you can never be sure, that the Ess-Component is already started or did not change since last run.